var/spack/repos/builtin/packages/apex/package.py

```diff
@@ -176,9 +178,6 @@ def cmake_args(self):
         if "+jemalloc" in spec:
             args.append("-DJEMALLOC_ROOT={0}".format(spec["jemalloc"].prefix))
 
-        if "+boost" in spec:
-            args.append("-DBOOST_ROOT={0}".format(spec["boost"].prefix))
-
```

Comment:
Would this work for old versions?

Reply:
Yes, because it was only needed for older (non-C+11 compliant) intel compilers on KNL architectures specifically. The rest of the code is up to C++17 now anyway. No reason to keep that around.